doc/rados/operations/control.rst

```diff
@@ -44,14 +44,53 @@ or not it is in the quorum. ::
 Authentication Subsystem
 ========================
 
-To add a keyring for an OSD, execute the following::
+Add auth info for ``entity`` from input file, or random key if no input is given, and/or any caps specified. ::
```

Comment:
i don't think duplicating the section of https://github.com/ceph/ceph/blob/master/doc/man/8/ceph.rst#auth would help. instead you can just add a hyper link point to it.